packages/flutter/lib/src/widgets/scroll_controller.dart

```diff
@@ -142,6 +142,10 @@ class ScrollController extends ChangeNotifier {
   ///
   /// The duration must not be zero. To jump to a particular value without an
   /// animation, use [jumpTo].
+  ///
+  /// Note that when calling [animateTo] in widget tests, `await`ing the
```

Comment:
nit: as per style guide don't use "note that".

Reply:
Done.